Code/GraphMol/catch_chirality.cpp

```diff
@@ -299,6 +319,12 @@ TEST_CASE("isAtomPotentialTetrahedralCenter", "[unittest]") {
       CHECK(!Chirality::detail::isAtomPotentialTetrahedralCenter(
           mol->getAtomWithIdx(6)));
     }
+    {
+      auto mol = "O[P]([O-])(=O)OC"_smiles;
+      REQUIRE(mol);
+      CHECK(Chirality::detail::isAtomPotentialTetrahedralCenter(
+          mol->getAtomWithIdx(1)));
+    }
```

Comment:
Hmm, ok. Maybe not for that one, but `O[C@H]([O-])F` does end up with a CIP code assigned

Reply:
Hum, yes, the new labeler does label it... but the original "centres" program doesn't: it doesn't flag the C as stereogenic.

Reply:
Ok, InChI agrees with you that neither the charge here nor that in N[C@]([NH3+])(F))Cl creates a chiral center. I will remove that.

To this specific comment though: the atom is still a _potential_ stereogenic center (which is what `isAtomPotentialTetrahedralCenter()` is testing) since it has degree 4. Once I fix the problem with charges it will no longer be identified as a result by `findPotentialStereo()` though.